packages/redux/src/locale/reducer.ts

```diff
@@ -58,20 +58,20 @@ const countryCode = (
 
 const cities = (
   state = INITIAL_STATE_LOCALE.cities,
-  action: ActionFetchCountryCities,
+  action: ActionFetchCountryStateCities,
```

Comment:
Shouldn't this be `AnyAction` as you did in https://github.com/Farfetch/blackout/pull/460 ?